packages/package_velvet_1_2_10/tool_dependencies.xml

```diff
@@ -0,0 +1,31 @@
+<tool_dependency>
+    <package name="velvet" version="1.2.10">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://github.com/dzerbino/velvet/archive/9adf09f7ded7fedaf6b0e5e4edf9f46602e263d3.zip</action>
```

Comment:
That's exactly what I did. Personal opinion but as per @bgruening's comment, I trust an SHA1 sum from a git repo a lot more than a mutable file in someone's homedir.